run.py

```diff
@@ -102,11 +102,18 @@ def main():
     docker_arch = args.platform or ("linux/amd64/v2" if branch == "9.0" else "linux/amd64")
 
     docker_args = [RUNNER, "run", "-i", "-t",
-                   "-u", "builder",
                    "--platform", docker_arch,
                    ]
     if is_podman(RUNNER):
-        docker_args += ["--userns=keep-id", "--security-opt", "label=disable"]
+        # With podman we use the `--userns` option to map the builder user to
+        # the user on the system.
+        docker_args += [f"--userns=keep-id:uid=1000,gid=1000", "--security-opt", "label=disable"]
+    else:
+        # With docker, we modify the builder user in the entrypoint to match the
+        # uid:gid of the user launching the container, and then continue with
+        # the builder user thanks to gosu.
+        docker_args += ["-e", f'BUILDER_UID={os.getuid()}', "-e", f'BUILDER_GID={os.getgid()}']
```

Comment:
a reader might ask "why gosu and and just su"

Reply:
Then he/she will quickly find on the internet why :-)

I usually try to only document what is specific to the project.
Do you think there is something specific about gosu in that context that should be commented?

Reply:
Finding out about `gosu` did not teach me the answer.  I had to go to the *docker* doc to find out why in a docker context they recommend using `gosu`.  We need at least a pointer to that docker doc.